docs/en/writing-running-appium/caps.md

```diff
@@ -62,7 +62,7 @@ These Capabilities are available only on Android-based drivers (like
 |----|-----------|-------|
 |`appActivity`| Activity name for the Android activity you want to launch from your package. This often needs to be preceded by a `.` (e.g., `.MainActivity` instead of `MainActivity`). By default this capability is received from the package manifest (action: android.intent.action.MAIN , category: android.intent.category.LAUNCHER) |`MainActivity`, `.Settings`|
 |`appPackage`| Java package of the Android app you want to run. By default this capability is received from the package manifest (@package attribute value)|`com.example.android.myApp`, `com.android.settings`|
-|`appWaitActivity`| Activity name/names, comma separated, for the Android activity you want to wait for. By default the value of this capability is the same as for `appActivity`. You must set `appWaitActivity` to wait for activities if your capability has `appActivity` and `appPackage`. The behaviour switch on [`startAppOptions.waitActivity`](https://github.com/appium/appium-adb/blob/master/lib/tools/apk-utils.js#L119). |`SplashActivity`, `SplashActivity,OtherActivity`, `*`, `*.SplashActivity`|
+|`appWaitActivity`| Activity name/names, comma separated, for the Android activity you want to wait for. By default the value of this capability is the same as for `appActivity`. You must set `appWaitActivity` to wait for activities if your capability has `appActivity` and `appPackage`. |`SplashActivity`, `SplashActivity,OtherActivity`, `*`, `*.SplashActivity`|
```

Comment:
I'd rather say `set it to the very first focused application activity name in case it is different from the one, which is set as appActivity `